high-level-tests/e2e/cypress/fixtures/users.json

```diff
@@ -65,5 +65,14 @@
     "username": "user.shouldChangePassword1234",
     "password": "$2b$05$Ryd6vVDoubvN9MQmSGmfY.tscxdmWhu1g8NOrHI4dveemziMof6Qm",
     "shouldChangePassword": true
+  },
```

Comment:
Je propose comme titre de commit, pour rester cohérent
Add e2e test for new terms-of-service **approval** => Add e2e test for new terms of service **validation**
(ou acceptance, à la rigueur, comme l'API)